src/app/cakes/cake.interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface Cake {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.